setup.py

```diff
@@ -1,53 +1,2 @@
-from setuptools import setup
-from distutils.core import Extension
-from distutils.command.build import build
-import os
 from subprocess import call
-import multiprocessing
-from glob import glob
-
-nxppy = Extension('nxppy',
-                    sources = ['Mifare.c', 'nxppy.c'],
-                    extra_compile_args=['-O1'],
-                    extra_link_args=['-lwiringPi']
-)
-
-class build_nxppy(build):
-    def run(self):
-        def compile():
-            call( './get_nxpRdLib.sh', shell=True )
```

Comment:
This file should remain intact, as it's required to install nxppy from PyPI (i.e. "pip install nxppy").